camply/config/notification_config.py

```diff
@@ -68,6 +68,18 @@ class PushbulletConfig:
     API_TOKEN = getenv("PUSHBULLET_API_TOKEN", None)
 
 
+class TwilioConfig:
+    """
+    Twilio Notification Config Class
+    """
+
+    ACCOUNT_SID = getenv("TWILIO_ACCOUNT_SID", None)
+    AUTH_TOKEN = getenv("TWILIO_AUTH_TOKEN", None)
+    SOURCE_NUMBER = getenv("TWILIO_SOURCE_NUMBER", None)
+    # comma separated set of phone numbers
+    DEST_NUMBERS = getenv("TWILIO_DEST_NUMBERS", None)
```

Comment:
Love this - what an amazing feature! Give me some time to check this out and I am 💯 on board with this.